graphkb/constants.py

```diff
@@ -26,6 +74,7 @@
     ('biological', ['functional effect', 'tumourigenesis', 'predisposing']),
 ]
 
+AMBIGUOUS_AA = ['x', '?', 'X']
```

Comment:
This might be overkill, be we could add 'B' and 'Z' as ambiguous amino acids: https://www.ncbi.nlm.nih.gov/Class/MLACourse/Modules/MolBioReview/iupac_aa_abbreviations.html

Reply:
I think it's overkill.
No one uses these anymore, because no one does acid degradation to sequence proteins anymore.
(When dissolved by acid Asn -> Asp and Gln -> Glu, making them indistiguishable)